pkgs/top-level/all-packages.nix

```diff
@@ -14086,7 +14086,7 @@ with pkgs;
       num =
         if (with stdenv.targetPlatform; isVc4 || libc == "relibc") then 6
         else if (stdenv.targetPlatform.isAarch64 && stdenv.isLinux) then 9
```

Comment:
```suggestion
```

I think it should be good to go. If not, we can do #211007 instead.

Reply:
Further testing reveals we're not ready for this unfortunately.

Reply:
This branch wouldn't be revaled, it only ran on x86_64-linux.

@vcunat, since x86_64 seems good to go, would it make sense to include this suggestion and test aarch64-linux with gcc12?

Reply:
I am testing this branch on aarch64-linux right now, it's quite unlikely to work though.

Reply:
> I am testing this branch on aarch64-linux right now, it's quite unlikely to work though.

Indeed, I don't think it will work before #209063...

Reply:
Thanks for testing, I don't use aarch64-linux so can't help much.

If there are still rough edges then it is probably best to keep that aarch64 change as a separate commit.

Reply:
Will mark this thread as resolved.

Reply:
Yeah it gives essentially the same error it does for GCC 11. Please don't delay the merge of this commit for this reason.

We have made a lot of progress on this issue and have a (somewhat unsatisfactory) solution ready to go now once GCC 12 lands, but hope to get the underlying bootstrap issue fixed before 23.05. See [#208412](https://github.com/NixOS/nixpkgs/issues/208412) for some context. I wonder if some of the x86_64-linux failures are beginning to be caused by that same issue.